torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -909,6 +909,17 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
     return entries;
   }
 
+  c10::optional<FrameNodeId> currentFrameId() const {
```

Comment:
what is this stuff for ? tbh i havent super touched the interpreter so i might consider getting another eyes on this if there's anything tricky here

Reply:
it's basically for fingerprinting allocs with the node that they originate in/from. like i alluded to above, down here it serves a pretty trivial purpose (bookkeeping for tests) but 4 PRs up in the memorization sketch it necessary for inserting the nodes that hand out allocs captured by memory_observer